addContact.html

```diff
@@ -0,0 +1,27 @@
+<div class="w3-container">
+
+    <form>
+        <h3 ng-show="edit">Add New Contact:</h3>
+        <h3 ng-show="!edit">Edit Contact:</h3>
+        <label>First Name:</label>
+        <input class="w3-input w3-border" type="text" ng-model="currentContact.fName" placeholder="First Name">
+        <br>
```

Comment:
this should be 'br /' instead of "br", the Firefox console keeps complaining about a missing </br> :slightly_smiling_face: